elimu_yangu/leaguetable/views.py

```diff
@@ -300,14 +298,17 @@ def get_context_data(self, geo_id1, geo_id2):
 
             level, code = geo_id2.split('-', 1)
             page_context['geo2'] = geo_data.get_geography(code, level)
-        except (ValueError, LocationNotFound):
+        except (ValueError, Exception):
             raise Http404
 
         return page_context
 
 def get_overall_topschools(year, geo_level, geo_code, session):
     schools = {}
     # Choosing sorting option
+    print '\n\n\n\n\n'
+    print Base.metadata.tables
+    print '\n\n\n\n\n'
```

Comment:
I think you've forgotten about this.

Reply:
I was still debugging elimu_yangu, will delete when I am done